### PR TITLE
chore: update std for TypeScript 4.3 changes

### PR DESCRIPTION
**Pull request by kitsonk**
This will require canary once denoland/deno#9960 is merged until Deno 1.11 is released.

### CODE REVIEW

**Comment by kt3k** [comment]
@kitsonk Thanks!

File: io/buffer.ts [hunk 1]
```diff
@@ -168,15 +167,14 @@ export class Buffer {
...
-  /** Grows the buffer's capacity, if necessary, to guarantee space for
+  } /** Grows the buffer's capacity, if necessary, to guarantee space for
    * another `n` bytes. After `.grow(n)`, at least `n` bytes can be written to
    * the buffer without another allocation. If `n` is negative, `.grow()` will
    * throw. If the buffer can't grow it will throw an error.
    *
    * Based on Go Lang's
    * [Buffer.Grow](https://golang.org/pkg/bytes/#Buffer.Grow). */
+
   grow(n: number): void {
```
Comment (ry) [inline io/buffer.ts:178]: Intentional?

File: io/buffer.ts [hunk 2]
```diff
@@ -168,15 +167,14 @@ export class Buffer {
     this.#off = 0;
     this.#reslice(Math.min(m + n, MAX_SIZE));
     return m;
-  };
-
-  /** Grows the buffer's capacity, if necessary, to guarantee space for
+  } /** Grows the buffer's capacity, if necessary, to guarantee space for
```
Comment (lucacasonato) [inline io/buffer.ts:170]: This seems fishy.

Comment (kitsonk) [reply io/buffer.ts:170]: `deno fmt` mangled it, I know how to reproduce it so will open an issue.

File: io/buffer.ts (hunk 1 above)
Comment (kitsonk) [reply io/buffer.ts:178]: No, `deno fmt` mangled it, I know how to reproduce it so will open an issue.

**Comment by wperron** [comment]
we're hitting the 30m timeout :expressionless:

**Comment by kt3k** [comment]
timeout in CI seems being caused by the bug of `--coverage` option https://github.com/denoland/deno/issues/10788

**Comment by kt3k** [comment]
I disabled the coverage settings temporarily with a TODO comment mentioning https://github.com/denoland/deno/issues/10788

**Review by wperron — APPROVED** [review]

**Pull request merged**